### PR TITLE
fix(settings): ensure scroll on tiny screens

### PR DESCRIPTION
**Pull request by H4ad**
## Pull Request

---

The Roles page was not adding a scroll when the screen is tiny.

Before:

https://github.com/user-attachments/assets/ab4e65d3-f6bf-4dbe-92aa-8ab8ae487ec4

After:

https://github.com/user-attachments/assets/1e2b9bb2-d23f-4795-bec0-61302622b887

Related to https://github.com/SigNoz/platform-pod/issues/2529

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [x] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

- Tests added/updated: No
- Manual verification: Yes
- Edge cases covered: Yes

---

- Blast radius: Settings - Roles / Members
- Potential regressions:
- Rollback plan:

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Bug Fix |
| Description | Ensure the pages for members and roles has proper handling for tiny screens. |

---

### 📋 Checklist
- [x] Tests added or explicitly not required
- [x] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

### CODE REVIEW

File: frontend/src/container/MembersSettings/MembersSettings.styles.scss [hunk 1]
```diff
@@ -4,6 +4,24 @@
 	gap: var(--spacing-8);
 	padding: var(--padding-4) var(--padding-2) var(--padding-6) var(--padding-4);
 	height: 100%;
+	overflow-y: auto;
+
+	&::-webkit-scrollbar {
```
Comment (ahrefabhi) [inline frontend/src/container/MembersSettings/MembersSettings.styles.scss]: can we use the below mixin for scrollbar styles https://github.com/SigNoz/signoz/blob/949d18f0283fefb7c07df7fe9a811e5ece0ba38e/frontend/src/styles/_scrollbar.scss

Comment (H4ad) [reply frontend/src/container/MembersSettings/MembersSettings.styles.scss]: Updated, 

Image

**Review by ahrefabhi — APPROVED** [review]

**Review by YounixM — APPROVED** [review]
